examples/test/project/Main.tscn

```diff
@@ -8,17 +8,15 @@ script = ExtResource("1_nbra5")
 [node name="Panel" type="Panel" parent="."]
 
 [node name="Label" type="TestD" parent="Panel"]
-number = 24
-long_node_path = NodePath("One/Looooong/Incredibly/Unbroken/Node/Path/Label")
```

Comment:
This was probably due to broken importc before, this change will probably should be reverted

Reply:
Strangely enough it is not problem of importc coz godot does it by itself
Tried with both bindings and importc version, gonna need to be a bit more investigation